casa/IO/MultiFileBase.cc

```diff
@@ -78,9 +113,39 @@ namespace casacore { //# NAMESPACE CASACORE - BEGIN
 
   MultiFileBase::~MultiFileBase()
   {
+    // Note: do not call flush() here, otherwise the virtual function
+    // doFlushFile in the already destructed base class is called
```

Comment:
Which base class does this refer to? This class has no bases.

Reply:
Should be derived instead of base.